crates/bevy_transform/src/components/global_transform.rs

```diff
@@ -5,7 +5,8 @@ use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_math::{Affine3A, Mat4, Quat, Vec3, Vec3A};
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 
-/// Describe the position of an entity relative to the reference frame.
+/// Describe the absolute position of an entity in space,
```

Comment:
I agree, I've thought about it too, and it's used on the first line of the doc too, actually translation/rotation/scale are never really explained.
But I don't have a good enough replacement except enumerating the three (with translation) which is a bit annoying to put every time.
"positioning" might be a bit better as it implies rotation too (in my mind at least)?
I could use "pose" by defining it at the beginning (which I think I should do anyway)? Also "The term “pose” is largely synonymous with the term “transform”, but a transform may often include scale, whereas pose does not.".
Maybe just use the enumeration at first then just "transform" but it might be confusing?